config/locales/views.sbv.de.yml

```
@@ -7,6 +7,7 @@ de:
   global:
     last_change: "Letzte Änderung: %{date}"
   group:
+    festival_tab: Festival
```

```suggestion
    festival_tab: Festivals
```

The other tabs that lead to a list use the plural version of the word. SUISA is an exception to this, because it is the name of the government agency. :wink: